README.md

```diff
@@ -1,65 +1,24 @@
-# Primaza
+# :knot: Primaza
 
-## Multi Cluster Service Consumption Framework
+Primaza is a multi-cluster Service Consumption Framework.
+Primaza wants to be namespace-scoped and will not required any resource at cluster level.
```

Comment:
suggest to change this one: Primaza is namespace-scoped and does not require any resource at cluster level.

Reply:
We definitely require CRDs, which are cluster scoped, and our webhooks are (currently) cluster-scoped as well.  Maybe we should mention this?

Reply:
Added mention to CRDs. We also already have a [story for removing webhooks](https://issues.redhat.com/browse/APPSVC-1363), so I'd say we can omit them here.

Reply:
look good to me